README.md

```diff
@@ -28,42 +28,42 @@ issue / pull request should be filled on the reference repository.
 
 ## Contributions
 
-... are welcome :smile:. For details, look at
+...are welcome. :smile: For details, look at
 [CONTRIBUTING.md](/CONTRIBUTING.md).
 
-## Dependencies
-| **Name** | **Version** |
-|----------|-------------|
-| libsodium | >=1.0.13 |
-
 ## Building
-Fairly simple. You'll need [Rust] >= 1.26.0 and [libsodium].
+Fairly simple. First, install [Rust] >= 1.30.1.
```

Comment:
Do we also need clang now?

Reply:
No.

Reply:
ok, but gcc should be required, shouldn't it? https://github.com/alexcrichton/cc-rs#compile-time-requirements

Reply:
Rust actually asks you to install Build Tools for Visual Studio on Windows.

Reply:
Don't you think it should be mentioned here?